### DISCUSSION TITLE
Benchmarking Numbers

### DISCUSSION THREAD

**Discussion by shubhamguptadream11** [callstack/react-native-bottom-tabs#72] [General]
@okwasniewski Do we have any plan to publish benchmarking numbers for Native Bottom tabs as well, comparing this with JS bottom tabs.

This will help people to make judicial decision on adopting Native bottom tabs.

Benchmarking numbers may include:
- Tab switch time using an automated script.
- Mount time of bottom tabs.
- Any other performance number?

**Comment by okwasniewski** [comment]
Hey, 

Yeah for sure! This would be awesome. 

I just haven't found time to work on this yet. 

Regarding numbers, I think mount time + switch is the most important for now 👍🏻
